linebot/models/template.py

```diff
@@ -100,7 +100,10 @@ class ButtonsTemplate(Template):
     Template message with an image, title, text, and multiple action buttons.
     """
 
-    def __init__(self, text=None, title=None, thumbnail_image_url=None, actions=None, **kwargs):
+    def __init__(self, text=None, title=None, thumbnail_image_url=None,
```

Comment:
Would you like to set the default value to `None`?

`image_aspect_ratio=None, image_size=None, image_background_color=None`

(I do not think that, ) if the default value of API changes, it will be necessary to modify the SDK as well.

Reply:
I agree.

However, currently in model/base.py, function as_json_dict. For those keys with value None will produce "" value when encoded as json.
I'm not sure if sending empty value with keys like `imageSize="", imageAspectRatio=""` will cause any problem (such as overwrite server-side default value.)

Anyway I'll update the pull request. Thanks.

Reply:
Thank you for agreeing. :D

Humm...
Currently I think `None` value is serialized as `null`.
ref:
https://github.com/line/line-bot-sdk-python/blob/master/tests/models/test_base.py#L32-L68

( And server-side uses default when value is `null`. )